app/controllers/dashboards_controller.rb

```diff
@@ -45,6 +45,9 @@ def push
 
       if pv.persisted?
         flash.notice = 'Production data updated'
+
+        # Build Sitemap and notify search engines
+        GibctSiteMapper.new(ping: true)
```

Comment:
Approximately how long does this process take? Production publishing is fast right now, which is great; we don't want to lose responsiveness if we can help it. If it's a lengthy process maybe we ought to make it asynchronous.

Reply:
Its rather quick, not even noticeable.